doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -261,6 +261,10 @@ Amazon Sidewalk samples
 Bluetooth samples
 -----------------
 
+* Added support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the following samples:
```

Comment:
Consider aligning release note format with existing notes to improve consistency. We can also mention that L05 and L10 are emulated on L15 SoC.
For example, see nRF Desktop release note:

```
  * Application configurations for the nRF54L05 and nRF54L10 SoCs (emulated on the nRF54L15 DK).     
    The configurations are supported through ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
```